client/vat/form-modal/index.tsx

```diff
@@ -25,7 +29,7 @@ const VatFormModal = ( {
 			shouldCloseOnClickOutside={ true }
```

Comment:
Should we do `isDismissible={ true }` instead since there's not a _Cancel_ button? Just so there's some sort of UI to dismiss the modal?

Reply:
No worries, GH is bit quirky when commenting on code that hasn't been touched in the current PR. I've removed the `isDismissible={ false }` prop, as modals default that to true, and added a test in https://github.com/Automattic/woocommerce-payments/pull/4168/commits/57402b44d14c57ced68247f9d563483b476036b7.